rllib/core/learner/torch/torch_learner.py

```diff
@@ -43,7 +43,7 @@
 torch, nn = try_import_torch()
 
 if torch:
-    from ray.air._internal.torch_utils import get_devices
+    from ray.air._internal.torch_utils import get_device
```

Comment:
Can you explain the difference? Would both methods work or was this a bug in our RLlib code?